data_model/src/isi.rs

```diff
@@ -615,31 +610,11 @@ mod transparent {
         }
     }
 
-    impl Mint<u32, Asset> {
-        /// Constructs a new [`Mint`] for an [`Asset`] of [`Quantity`] type.
-        pub fn asset_quantity(quantity: u32, asset_id: AssetId) -> Self {
+    impl Mint<Numeric, Asset> {
+        /// Constructs a new [`Mint`] for an [`Asset`] of [`Numeric`] type.
+        pub fn asset_numeric(numeric: Numeric, asset_id: AssetId) -> Self {
```

Comment:
Maybe make it accept `Into<Numeric>` for better ergonomics? This will allow to not use the `numeric!` macro most of the time

Reply:
Implemented, but not actually sure it makes api better...